bin/compile

```diff
@@ -14,6 +14,15 @@ env_dir=$3
 
 cd $build_dir
 
+[ -f "$build_dir/yarn.lock" ] && YARN=true || YARN=false
```

Comment:
@brunoarueira could you unpack this line for me? I can't decide if it's supposed to be an assignment or a condition test...